include/kyosu/functions/bessel_h.hpp

```diff
@@ -0,0 +1,151 @@
+//======================================================================================================================
+/*
+  Kyosu - Complex Without Complexes
+  Copyright: KYOSU Contributors & Maintainers
+  SPDX-License-Identifier: BSL-1.0
+*/
+//======================================================================================================================
+#pragma once
+#include "eve/traits/as_logical.hpp"
+#include <kyosu/details/callable.hpp>
+#include <kyosu/bessel.hpp>
+
+
+namespace kyosu
+{
+
+  template<typename Options>
+  struct bessel_h_t : eve::strict_elementwise_callable<bessel_h_t, Options, eve::spherical_option, eve::cylindrical_option, kind_1_option, kind_2_option>
+  {
+    template<eve::scalar_value N, typename Z>
+    requires(concepts::real<Z> || concepts::cayley_dickson<Z>)
+      KYOSU_FORCEINLINE constexpr auto  operator()(N const& n, Z const & z) const noexcept
+    {
+      constexpr int Kind = Options::contains(kind_2) ? 2 : 1;
```

Comment:
why not having a static constexpr field inside the Kind option ?

Reply:
preferef that solution less verbose

Reply:
Yet you repeat it at least 3 times

Reply:
and so